packaging/installer/methods/offline.md

```diff
@@ -34,7 +34,7 @@ curl -s https://api.github.com/repos/netdata/netdata/releases/latest | grep "bro
 curl -s https://api.github.com/repos/netdata/netdata/releases/latest | grep "browser_download_url.*txt" | cut -d '"' -f 4 | wget -qi -
 
 # Netdata dependency handling script
-curl -s https://raw.githubusercontent.com/netdata/netdata-demo-site/master/install-required-packages.sh | wget -qi -
+wget -q - https://raw.githubusercontent.com/netdata/netdata-demo-site/master/install-required-packages.sh
```

Comment:
The correct link to this script is actually:

https://raw.githubusercontent.com/netdata/netdata/master/packaging/installer/install-required-packages.sh

Can you amend this?

Reply:
Sure